lib/context_adapter_helper.js

```diff
@@ -205,6 +217,71 @@ function isAttributeIncluded(attributes, attributeName) {
 }
 
 /**
+ * Validates the attribute values received
+ * @param {object} context The context for logging
+ * @param {string} entityType The entity type (Button or Service)
+ * @param {Array} attributes The attributes array
+ * @return {*} True if the attribute value is valid, false otherwise
+ */
+function isValidAttributeValues(context, entityType, attributes) {
```

Comment:
When the validations are that complex, it may be a good idea to split the function in multiple validation functions, one for each type of validation. This particular one is near the limit, I think. I leave it up to you if you think is appropriate, but, if you are planning to introduce more validations, I would split it to improve readability.